submissions/AndriyBerezovskiy/hooli-style-popup/index.html

```diff
@@ -0,0 +1,234 @@
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/chat.svg" 
+                          alt=""
+                        />
+                        <span>Chat</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/qrcode.svg" 
+                          alt=""
+                        />
+                        <span>Qrcode</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/database.svg" 
+                          alt=""
+                        />
+                        <span>Database</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/gift.svg" 
+                          alt=""
+                        />
+                        <span>Gift</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/printer.svg" 
+                          alt=""
+                        />
+                        <span>Printer</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" 
+                          src="./img/translate.svg" 
+                          alt=""
+                        />
+                        <span>Translate</span>
+                      </a>
+                    </li>
+                  </ul>
+                  <p class="popup__text">
+                    <a
+                      href="https://github.com/kottans/frontend/blob/2022_UA/contents.md"
+                      >Even more from Kottans</a
+                    >
+                  </p>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li>
+            <a class="header__link" href="#">
+              <img src="./img/bell.svg" alt="go to the notifications page" />
+            </a>
+          </li>
+          <li>
+            <a class="header__link" href="#">
+              <img src="./img/user.svg" alt="go to user information" />
```

Comment:
What about these images?

Reply:
I thought that if the icon is not signed at all, then `alt` is needed. But based on your previous comment about icons being decorative images, it appears that all icons don't need `alt`. Fixed it